be/src/exec/pipeline/scan_operator.cpp

```diff
@@ -110,7 +114,7 @@ bool ScanOperator::has_output() const {
 }
 
 bool ScanOperator::pending_finish() {
-    DCHECK(_is_finished);
+    //DCHECK(_is_finished);
```

Comment:
Why comment it?

Reply:
pending_finish means that ScanOperator has already produced sufficient chunks, but it has an pending io task executed by io threads, so ScanOperator can not be destructed immediately, it should wait until pending io tasks has finishe. in such situtations, ScanOperator::_is_finished must be true.